src/get-started/java-binding.md

```diff
@@ -180,11 +198,23 @@ current storage state when the transaction is executed. Exonum passes `Fork`
 as an argument – a view that allows performing modifying operations. A service
 schema object can be used to access data collections of this service.
 
+Also, `Transaction#execute` method may throw `TransactionExecutionException`
+which contains a transaction error report. This feature allows users to notify
+Exonum about an error in a transaction execution whenever one occurs.
+The service checks the preconditions before executing a transaction and either
```

Comment:
I think it is not accurate enough. This very *method* may check the preconditions and may either throw or return, not "The service".